chromium_src/chrome/browser/ui/views/profiles/avatar_toolbar_button_delegate.cc

```diff
@@ -1,35 +0,0 @@
-/* Copyright (c) 2020 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#include "chrome/browser/browser_process.h"
-#include "chrome/browser/profiles/profile_attributes_entry.h"
-#include "chrome/browser/profiles/profile_avatar_icon_util.h"
-#include "chrome/browser/profiles/profile_manager.h"
-
-#include "src/chrome/browser/ui/views/profiles/avatar_toolbar_button_delegate.cc"
-
-BraveAvatarToolbarButtonDelegate::BraveAvatarToolbarButtonDelegate(
-    AvatarToolbarButton* button,
-    Profile* profile)
-    : AvatarToolbarButtonDelegate(button, profile), profile_(profile) {}
-
-AvatarToolbarButton::State BraveAvatarToolbarButtonDelegate::GetState() const {
-  AvatarToolbarButton::State state = AvatarToolbarButtonDelegate::GetState();
-  if (state == AvatarToolbarButton::State::kNormal) {
-    ProfileAttributesEntry* entry =
-        g_browser_process->profile_manager()
-            ->GetProfileAttributesStorage()
-            .GetProfileAttributesWithPath(profile_->GetPath());
-    if (entry &&
-        entry->GetAvatarIconIndex() == profiles::GetPlaceholderAvatarIndex()) {
-      return AvatarToolbarButton::State::kNormal;
-    }
-  }
-  return state;
-}
```

Comment:
It looks dead code to me, which always returns the parent class's implementation. If we remove this, we don't need this chromium_src/ override becuase we don't need that "virtual" overriding.